### ISSUE TITLE
Omega mapping/atmos issues

### ISSUE THREAD

**Issue by 81Denton** [tgstation/tgstation#36954]
I'll just write these in one ticket; easy fixes that would clutter the issues tab.

Mapping:
1) There are no solars. This pretty much guarantees a round end if the SME blows up.
2) The southern engineering windows are r-glass instead of r-plasmaglass. This is bad since a decent plasma fire will break them quickly.
3) All restricted areas can be reached by smashing r-glass windows. Armory, vault, captain's office, oof.

Atmos:
1) There are many areas with vents/scrubbers, but no air alarms. For example: genetics, the small hallways between engineering/atmos, the gravgen lobby.
2) Many areas either only have a vent or a scrubber. Armory, sec equipment room, gravgen, brig cells.

Not a bug, but could still be improved:
1) Most departments have plastic flaps, even though the station has no MULEbots.
2) Vault has no turrets and can be entered by smashing r-glass windows. AI/borg upload computers don't have r-glass windows and windoors in front of them either.
3) Vault, armory and AI sat don't have motion sensitive cams. Probably not a huge deal, considering how small the station is.
4) Medbay has no defib mount (no CMO locker) and no bloodpacks.
5) Botany and the kitchen aren't next to each other; there aren't disposals to send vegetables to the chef either.
6) There are no escape pods.

tl;dr Label: FUCK and also please replace Omega with Pubby until people have had time to fix all of this

**Comment by 81Denton** [comment]
Alright, updated it with the issues fixed in #36964.

**Comment by miniusAreas** [comment]
This supposed to be closed @81Denton ?

**Comment by 81Denton** [comment]
Nah, the issues still exist. There is just nobody who wants to put time into fixing Omega.
